sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -1706,14 +1706,11 @@ object SupportedOpsDocs {
     println()
     println("# General limitations")
     println("## `Decimal`")
-    println("The `Decimal` type in Spark supports a precision")
-    println("up to 38 digits (128-bits). The RAPIDS Accelerator in most cases stores values up to")
-    println("64-bits and will support 128-bit in the future. As such the accelerator currently only")
-    println(s"supports a precision up to ${DType.DECIMAL64_MAX_PRECISION} digits. Note that")
-    println("decimals are disabled by default in the plugin, because it is supported by a relatively")
-    println("small number of operations presently. This can result in a lot of data movement to and")
-    println("from the GPU, slowing down processing in some cases.")
-    println("Result `Decimal` precision and scale follow the same rule as CPU mode in Apache Spark:")
+    println("The `Decimal` type in Spark supports a precision up to 38 digits (128-bits). ")
+    println("The RAPIDS Accelerator supports 128-bit starting from version 21.12 and decimals are enabled by default.")
```

Comment:
Do we have a limit of 100 characters in a line?

Reply:
Fixed